components/authz-service/server/v2/project.go

```diff
@@ -19,7 +20,7 @@ import (
 	api "github.com/chef/automate/api/interservice/authz/v2"
 	automate_event "github.com/chef/automate/api/interservice/event"
 	"github.com/chef/automate/components/authz-service/config"
-	v2_constants "github.com/chef/automate/components/authz-service/constants/v2"
+	constants_v2 "github.com/chef/automate/components/authz-service/constants/v2"
```

Comment:
Drive-by: the uses of "constants_v2" far outweighed "v2_constants" in the codebase so I changed those few of the latter to the former.